### PR TITLE
Microverse Portfolio: mobile menu

### PR DESCRIPTION
**Pull request by Fombi-Favour**
In this activity, we: 
- Modified the index.html file
- Modified the style.css file for the menu
- Added an index.js file for interactive style

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @Fombi-Favour ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Kindly ensure your coding partner contributes to this pull request since this is a pair programming activity.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Fombi-Favour** [comment]
Hello,
My partner contributed into the project and made commit.
Here is his commit [here!](https://github.com/Fombi-Favour/Microverse-portfolio/pull/10/commits/ac3339f61c666cbcdf519df81f9beb8d99da403d)

**Comment by Fombi-Favour** [comment]
My coding partner made some commits and here are those commits.
- [first commit](https://github.com/Fombi-Favour/Microverse-portfolio/commit/23d5f07f3f6887ede8db7ac5edcffeb1e2cb8bc6)
- [second commit](https://github.com/Fombi-Favour/Microverse-portfolio/commit/1103c65d7d0f554af962504e78fc7acd9c485a81)
- [third commit](https://github.com/Fombi-Favour/Microverse-portfolio/commit/0c1b01f8a9ba9a7c08c9891baea0e3b808ec0d54)
- [fourth commit](https://github.com/Fombi-Favour/Microverse-portfolio/commit/ac3339f61c666cbcdf519df81f9beb8d99da403d)

Please kindly consider this 🙏🙏

**Comment by ahmetbozaci** [comment]
### Hello @Fombi-Favour 
### _This is an additional review as a result of the second opinion request._ 
- Kindly don't make any changes if you submit a second opinion. You filled out the second opinion form after your partner make changes. Before the code reviewer reviews it your partner only one [commit](https://github.com/Fombi-Favour/Microverse-portfolio/pull/10/commits/ac3339f61c666cbcdf519df81f9beb8d99da403d). So code reviewer is right to invalidate it.
- [ ]  I see your partner made more commits so kindly submit it for review again.

___



### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by Fombi-Favour** [comment]
Before submitting request for code review, my coding partner made commit.
Here was his first commit before my submission [here!](https://github.com/Fombi-Favour/Microverse-portfolio/pull/10/commits/ac3339f61c666cbcdf519df81f9beb8d99da403d)

After that review, my coding partner and I were trying to solve the problem and there was no way until now. I tried to comment on his review but he didn't reply back.

Please kindly consider this commit 🙏🙏

**Comment by ahmetbozaci** [comment]
> Before submitting request for code review, my coding partner made commit. Here was his first commit before my submission [here!](https://github.com/Fombi-Favour/Microverse-portfolio/pull/10/commits/ac3339f61c666cbcdf519df81f9beb8d99da403d)
> 
> After that review, my coding partner and I were trying to solve the problem and there was no way until now. I tried to comment on his review but he didn't reply back.
> 
> Please kindly consider this commit praypray

- When working on a collaborative project, it is important to strive for equal contributions. However, this doesn't necessarily mean that the number of commits should be the same. **Your partner has already made more changes, I kindly request you submit your work for review again so that the code reviewer can begin evaluating the entire project.**
- Also ı recommend your partner fix the issue on the github for future projects because he/she isn't on the participant list too.

**Comment by Fombi-Favour** [comment]
I appreciate that. Thank you 🙏 🙏

File: index.html [hunk 1]
```diff
@@ -16,14 +16,17 @@
...
-          <img src="./assets/menu.png" alt="logo" />
+          <div class="nav-link" id="navLink">
+            <img src="assets/close.png" alt="close" onclick="hideMenu()" />
+            <ul>
+              <li id="Portfolio" onclick="hideMenu()"><a href="#portfolio">Portfolio</a></li>
+              <li id="About-me" onclick="hideMenu()"><a href="#about">About</a></li>
+              <li id="Contact-me" onclick="hideMenu()"><a href="#contact">Contact</a></li>
+            </ul>
+          </div>
+          <img src="./assets/menu.png" id="menu" alt="logo" onclick="showMenu()" />
```
Comment (Davidosky007) [inline index.html]: * Currently when I click on the hamburger icon on mobile it doesn't show the mobile menu dropdown, kindly fix this

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Fombi-Favour 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Basic JavaScript syntax used 👍 
* Events used 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @Fombi-Favour        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] linters are passing
- [x] Professional README 💯
- [x] Mobile Menu workimg as intended 🥇 
- [x] Nice UI💯 
 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**